DigitalLearningSolutions.Web/Startup.cs

```diff
@@ -164,7 +164,8 @@ public void Configure(IApplicationBuilder app, IMigrationRunner migrationRunner)
 
         private Task RedirectToLogin(RedirectContext<CookieAuthenticationOptions> context)
         {
-            var url = HttpUtility.UrlEncode(context.Request.Path);
+            var applicationPath = new Uri(config["AppRootPath"]).AbsolutePath.TrimEnd('/');
+            var url = HttpUtility.UrlEncode(applicationPath) + HttpUtility.UrlEncode(context.Request.Path);

```

Comment:
Minor, but I think it would make sense to do the url encoding after putting the url together:
`HttpUtility.UrlEncode(applicationPath + context.Request.Path)`